### PR TITLE
Backport Cover plaform from Core

### PR DESCRIPTION
**Pull request by tetienne**
- [ ]  Add slow speed service

<a href="https://gitpod.io/#https://github.com/iMicknl/ha-tahoma/pull/751"></a>

### CODE REVIEW

File: custom_components/tahoma/cover_entities/generic_cover.py [hunk 1]
```diff
@@ -1,68 +1,73 @@
...
 from homeassistant.components.cover import (
-    ATTR_POSITION,
     ATTR_TILT_POSITION,
     SUPPORT_CLOSE_TILT,
     SUPPORT_OPEN_TILT,
     SUPPORT_SET_TILT_POSITION,
     SUPPORT_STOP_TILT,
     CoverEntity,
 )
+from homeassistant.components.overkiz.entity import OverkizEntity
```
Comment (iMicknl) [inline custom_components/tahoma/cover_entities/generic_cover.py:15]:
```suggestion
from homeassistant.components.overkiz.entity import OverkizEntity
```

This is not a correct import :-).

Comment (tetienne) [reply custom_components/tahoma/cover_entities/generic_cover.py]: You saved me!

Comment (iMicknl) [reply custom_components/tahoma/cover_entities/generic_cover.py]: Hehe, I also added that ugly non relative import because of codestyle rules in core..

Comment (tetienne) [reply custom_components/tahoma/cover_entities/generic_cover.py]: Good to know, thx 👍

**Review by iMicknl — APPROVED** [review]
1:1 core code, so LGTM. Would you want to get low position cover back in? Or not yet?

**Comment by tetienne** [comment]
@iMicknl Sorry, I’ve enabled the auto merge. Yes, we should add back the low position service (but not the switch).

**Pull request merged**